src/ExtensionsSample/Program.cs

```diff
@@ -65,7 +66,8 @@ public static void Main(string[] args)
                 typeof(SendGridSamples),
                 typeof(TableSamples),
                 typeof(TimerSamples),
-                typeof(WebHookSamples));
+                typeof(WebHookSamples),
+                typeof(NotificationHubSamples));
```

Comment:
It isn't supposed to run by default (EasyTables and DocDb aren't there either). We want people to be able to hit F5 without a ton of setup. Then if they want to run these, they go add it (you have directions in the sample for how).